confd/vars/docker-compose.yaml

```diff
@@ -1,11 +1,16 @@
 ---
-#no_gui: true
+no_gui: true
 #no_kibana: true
-#no_floodlight_2: true
-#no_floodlight_stats: true
-#no_grpc_speaker: true
+no_floodlight_2: true
+no_floodlight_stats: true
+no_grpc_speaker: true
 #no_storm_ui: true
 #no_opentsdb: true
 #no_elasticsearch: true
 #no_logstash: true
-#no_grpc_stub: true
+no_grpc_stub: true
```

Comment:
I think disabling of these components by default is a bad idea. These flags are for developers. As we are still opensource I think we should torn everything on by default